migrations/00283_primary_episode_trigger.sql

```diff
@@ -0,0 +1,23 @@
+-- +goose Up
+
+
+CREATE OR REPLACE FUNCTION update_primary_episode() RETURNS TRIGGER AS $$
+BEGIN
+    IF (SELECT primary_episode_id FROM mediaitems WHERE id = NEW.mediaitem_id) IS NULL THEN
+        UPDATE mediaitems SET primary_episode_id = NEW.id WHERE id = NEW.mediaitem_id;
+    END IF;
+    RETURN NEW;
+END;
+$$ LANGUAGE plpgsql;
```

Comment:
You may need `-- +goose StatementStart` and `-- +goose StatementEnd` around the function def. If it works without, it's fine, jut wanted to make sure it's tested.